Setup portfolio mobile version

kwekubright opened the pull request:
### Description
In this feature I have done the following:

- Setup HTML&CSS linter
- Added header and headline sections to index.html
- Added style.css with flexbox rules for header and headline section
- Added social media icons and headline background image used in index.html

kwekubright committed:
Setup project

Add HTML&CSS linter, style.css, README.md and an index.html file

kwekubright committed:
Update index.html

Add header and a headline section

kwekubright committed:
Update style.css

Make header flex container with flex items and add background image to headline section.
Add headline-bg.png file

kwekubright committed:
Update index.html

Add social media list and social media icon.
Add header background image

kwekubright committed:
Update style.css

Add flexbox properties to social media list and position social media flex items.
Position header background image

kwekubright committed:
Update .gitignore

Add images/.D_Store system file to gitignore file

kwekubright committed:
Update .gitignore

Update .DS_Store definition in .gitignore

kwekubright committed:
Add screenshot.png

Add screenshot of setup

kwekubright committed:
Update README.md

Add screenshot to README

kwekubright committed:
Update style.css

Add unit to "0" values
Place pace before and after ">" selector

kwekubright committed:
Update style.css

Remove unit from "0" values

kwekubright committed:
Update index.html

Update index.html to include nav in the header area.

Add flexbox to nav in style.css

kwekubright committed:
Update style.css

Add space before "{" on line 38

kwekubright committed:
Delete .DS_Store

Delete .DS_Store. File is being ignored

OmarRamoun commented on index.html:
```diff
@@ -0,0 +1,61 @@
+  <body>
+    <header>
+      <nav class="header-nav wrap">
+        <h1 class="logo-text">Bright</h1>
```
- [ ] [REQUIRED] I think it would be better if you replace ♻️ the `<h1>` element with a `<a>` element for semantic purposes and also in order for the logo to be clickable (as users used to on the internet 🌐)

OmarRamoun commented on index.html:
```diff
@@ -0,0 +1,61 @@
+      <div class="headline-wrap">
+        <h2 class="welcome">I’m Bright Glad to see you!</h2>
+        <p class="headline-profile margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="headline-cta margin-top-12">Let's connect</a>
```
- [ ] [REQUIRED] Please, make sure that the subheading "Let's Connect" is a (`h2`, `h3`, ..., `h6`) and not an `<a>` (anchor element) ♻️. In order for your code to be more semantic.

OmarRamoun commented on index.html:
```diff
@@ -0,0 +1,61 @@
+        <a href="#" class="headline-cta margin-top-12">Let's connect</a>
+        <ul class="social-icons margin-top-12">
+          <li>
+            <a>
+              <img class="social twitter" src="images/icons/twitter-icon.png" alt="Twitter icon">
```
- [ ] [REQUIRED] I think it would be great if you add a dead link `href='#'` attribute to all the `<a>` tags for the different social icons to make them clickable 🔗

OmarRamoun commented on css/style.css:
```diff
@@ -0,0 +1,117 @@
+  background-color: #fff;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
```
- [ ] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.flex` class in multiple places.

- (your code)

```css
.header-nav {
  /* ...code */
  display: flex;
  /* ...code */
}

.headline-wrap {
  /* ...code */
  display: flex;
  /* ...code */
}
```

- suggestion

```css
.flex {
  display: flex;
}
```

OmarRamoun reviewed (changes requested):
# 🟡 Almost There! 🏃

## Hi @kwekubright  👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ Pull Request title and description are very descriptive
- ✔️ Use github Flow
- ✔️ descriptive commit messages.

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better 👍. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code 💯_

You can also consider:

- [ ] [OPTIONAL] Please, use a smaller size image for the screenshot of the project 🖼️ in the `README.md` file.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **Twitter**: [![@ramoun16](https://img.shields.io/twitter/follow/ramoun16?label=ramoun16&style=social)](https://twitter.com/ramoun16)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kwekubright committed:
Update index.html

Add <a> around logo to make it clickable
Add flex class on all flex containers for DRY purposes
Update style.css adding flex class and resize screenshot in README

kwekubright committed:
Merge branch 'setups' of github.com:kwekubright/portfolio into setups

kwekubright committed:
Update README.md

update screenshot height

kwekubright committed:
Update index.html

Replace <h1> with <a> for logo for semantic purposes

zainsadaqat reviewed (approved):
# Hello @kwekubright  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 

<img src="https://media3.giphy.com/media/3KC2jD2QcBOSc/giphy.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
